jetty-websocket/websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/WebSocketAdapter.java

```diff
@@ -34,7 +35,8 @@ public Session getSession()
 
     public boolean isConnected()
     {
-        return session.isOpen();
+        Session sess = this.session;
+        return (sess != null) && (sess.isOpen());
```

Comment:
I think just `return session != null && session.isOpen();` should be more readable. Creating `sess` here seems redundant.

Reply:
This is standard multi threading coding practice.
The value of `this.session` can actually change between the 2 evaluations (non-null check and isopen).
It could go from non-null to null between the two checks.